types/openlayers/index.d.ts

```diff
@@ -6783,7 +6784,7 @@ declare module ol {
         getFeaturesAtPixel(
             pixel: ol.Pixel,
             opt_options?: olx.AtPixelOptions
-        ): (Array<ol.Feature|ol.render.Feature>|null);
+        ): (Array<ol.Feature | ol.render.Feature> | null);
```

Comment:
I don't see any reason to change the code style. It would be better to keep it coherent with the rest of the file.

Reply:
ok, I'll change that